akka-remote/src/main/scala/akka/remote/RemoteWatcher.scala

```diff
@@ -32,7 +32,7 @@ private[akka] object RemoteWatcher {
   final case class UnwatchRemote(watchee: InternalActorRef, watcher: InternalActorRef)
 
   @SerialVersionUID(1L) case object Heartbeat extends HeartbeatMessage
-  @SerialVersionUID(1L) final case class HeartbeatRsp(addressUid: Int) extends HeartbeatMessage
+  @SerialVersionUID(1L) final case class HeartbeatRsp(addressUid: Long) extends HeartbeatMessage
```

Comment:
But, `ClusterRemoteWatcher` extends `RemoteWatcher` so if I watch another actor in another cluster node it will still use `Heartbeat/HeartbeatRsp`, no?